volatility3/framework/symbols/linux/__init__.py

```diff
@@ -830,3 +831,123 @@ def get_cached_pages(self) -> Iterator[interfaces.objects.ObjectInterface]:
             page = self.vmlinux.object("page", offset=page_addr, absolute=True)
             if page:
                 yield page
+
+
+class Tainting:
+    """Tainted kernel and modules parsing capabilities.
+
+    Relevant kernel functions:
+        - modules: module_flags_taint
+        - kernel: print_tainted
+    """
+
+    def __init__(
```

Comment:
I'd also expected these just to be function containers, rather than carrying state themselves.  We can have them carry state I guess, I'm just not sure the extra complexity/separation of parameters is worth it?